### PR TITLE
Persistence Dtos Add StoregDtos

### LINKED ISSUE #67: PERSISTENCE : Dtos StoragDtos

### PR DESCRIPTION
**Pull request by rustambekovich**
close #67

### CODE REVIEW

**Review by SamandarYigitaliyev — APPROVED** [review]
Hammasi to'g'ri

File: src/GreenSaleuz.Persistence/Dtos/StoragDtos/StoragCreateDto.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public string Name { get; set; } = string.Empty;
+    public string Description { get; set; } = string.Empty;
+    public string Region { get; set; } = string.Empty;
+    public string District { get; set; } = string.Empty;
+    public string Address { get; set; } = string.Empty;
+    public double AddressLatitude { get; set; }
+    public double AddressLongitude { get; set; }
+    public string Info { get; set; } = string.Empty;
+    public List<IFormFile> ImagePath { get; set; } = default!;
```
Comment (SamandarYigitaliyev) [inline src/GreenSaleuz.Persistence/Dtos/StoragDtos/StoragCreateDto.cs:15]: should accept a string, not a list

**Review by SamandarYigitaliyev — CHANGES_REQUESTED** [review]
Reload correctly

**Review by SamandarYigitaliyev — COMMENTED** [review]
correct

**Review by SamandarYigitaliyev — APPROVED** [review]
Correct

**Comment by MrOOI** [comment]
Closes #68

**Pull request merged**